hail/src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -199,13 +200,20 @@ object ExportVCF {
   def getAttributes(k1: String, k2: String, k3: String, attributes: Option[VCFMetadata]): Option[String] =
     getAttributes(k1, k2, attributes).flatMap(_.get(k3))
 
-  def apply(vsm: MatrixTable, path: String, append: Option[String] = None,
+  def apply(mt: MatrixTable, path: String, append: Option[String] = None,
     exportType: Int = ExportType.CONCATENATED, metadata: Option[VCFMetadata] = None) {
-    
-    vsm.requireColKeyString("export_vcf")
-    vsm.requireRowKeyVariant("export_vcf")
-    
-    val tg = vsm.entryType match {
+    ExportVCF(mt.value, path, append, exportType, metadata)
+  }
+
+  def apply(mv: MatrixValue, path: String, append: Option[String],
+    exportType: Int, metadata: Option[VCFMetadata]) {
+
+    val typ = mv.typ
+
+    mv.requireColKeyString("export_vcf")
```

Comment:
these are checked in Python. We can just assert here

Reply:
These functions are doing the assertion. I think leaving this is fine.

Reply:
I modified them to just do an assert.